.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
```

Comment:
While I understand your intent here, you're removing versions that the `composer.json` file says the library is compatible with: https://github.com/vim-php/phpctags/pull/73/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R7.